content/blog/2025-07-08-aligning-with-semver.md

```diff
@@ -0,0 +1,71 @@
+---
+authors: ["kokkos-team"]
+title: "Aligning Our Numbers: Kokkos Adopts Semantic Versioning with v5.0"
+date: "2025-07-08"
+tags: ["2025", "5.0"]
+thumbnail: img/blog/2025/semver-in-5x-series.png
+---
+
+
+**TL;DR: Starting with the Kokkos 5.x series, we're officially aligning with
+[Semantic Versioning (SemVer) 2.0.0](https://semver.org). This means our patch
+versions will now look like `5.X.Y` instead of `5.X.0Y`, getting rid of that
+leading zero that wasn't quite SemVer-compliant**
+
+For years, the Kokkos team has strived for consistency in our versioning, even
+when it meant sticking to some unconventional numbering. Since the `2.8.00`
+release in 2019, you might have noticed our patch versions featured a leading
+zero (e.g., `4.6.01`). This format was an attempt to maintain continuity with
+earlier practices.
+
+While we maintained this scheme out of a desire for consistency and concern for
+potentially breaking downstream scripts, it has led to many questions over the
+years. Frankly, most of us on the development team weren't fond of it either!
```

Comment:
I'd skip this paragraph or at least the last sentence since it doesn't really add anything and adds a (unnecessary) negative connotation. The first sentence, for the most part, just repeats the information in the last paragraph above and can be combined with it.